azext_edge/edge/providers/check/mq.py

```diff
@@ -1070,11 +1071,19 @@ def display_connector_info(
         )
```

Comment:
Should we add more specificity to the function name `display_connector_info()` ? Looks very focused on datalake connectors.

Reply:
We should, yeah - this is currently nested under `evaluate_datalake_connectors` as an inner function, and each connector type follows that pattern - but really each connector type should be refactored into its own module. I can address this in a follow-up PR